netclient/daemon/systemd.go

```diff
@@ -29,49 +29,28 @@ func SetupSystemDDaemon(interval string) error {
 		log.Println("couldnt find or create /etc/netclient")
 		return err
 	}
-
-	if !ncutils.FileExists("/usr/local/bin/netclient") {
-		os.Symlink("/etc/netclient/netclient", "/usr/local/bin/netclient")
-	}
-	if !ncutils.FileExists("/etc/netclient/netclient") {
-		err = ncutils.Copy(binarypath, "/etc/netclient/netclient")
+	//install binary
+	//should check if the existing binary is the corect version -- for now only copy if file doesn't exist
+	if !ncutils.FileExists("/usr/sbin/netclient") {
```

Comment:
"/usr/sbin/" should probably be a constant

Reply:
added constant... also updated netclient/daemon/macos.go